src/libraries/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -1,21 +1,21 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.IO;
 using System.Text;
 using Microsoft.DotNet.RemoteExecutor;
+using Microsoft.Win32;
 using Xunit;
 
 namespace System.Diagnostics.Tests
 {
     public class ProcessStandardConsoleTests : ProcessTestBase
     {
-        private const int s_ConsoleEncoding = 437;
+        private const int ConsoleEncoding = 437;
```

Comment:
Nit, looks like this could be a const local instead of a field. Also is there a better name such as DefaultConsoleCodePage ?